fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -68,22 +68,13 @@ def select_project(config)
     # Is this project a workspace?
     attr_accessor :is_workspace
 
-    # Should the output of xcodebuild commands be silenced?
-    attr_accessor :xcodebuild_list_silent
-
-    # Should we redirect stderr to /dev/null for xcodebuild commands?
-    # Gets rid of annoying plugin info warnings.
-    attr_accessor :xcodebuild_suppress_stderr
```

Comment:
As far as I investigated, those are added to improve Crashlytics's integration in the past https://github.com/fastlane/fastlane/pull/5491 but all integrations were deleted in https://github.com/fastlane/fastlane/pull/11601. It's not in use now.